project/Dependencies.scala

```diff
@@ -7,9 +7,9 @@ import scala.util.Try
 /* scalafmt: { maxColumn = 120 }*/
 
 object Dependencies {
-  val scala212 = "2.12.18"
-  val scala213 = "2.13.12"
-  val scala3 = "3.3.1"
+  val scala212 = sys.props.getOrElse("scala212.nightly", "2.12.18")
+  val scala213 = sys.props.getOrElse("scala213.nightly", "2.13.12")
```

Comment:
@SethTisue I have suffixed the system property with the scala version to make it possible to tweak any scala version, I assume this is not a problem as it's explicitly forwarded [there](https://github.com/SethTisue/community-build/blob/dcf2a870c2803a39006ae7a03ab7ddab2b65c29f/core/scalafix.conf#L10) so it can easily be changed?

Reply:
yup that's fine